### PR TITLE
DMN15-136 Align the usage of UnaryTests with decision table inputs

### PR DESCRIPTION
**Pull request by SimonRinguette**
Issue: [DMN15-135 Precision on Allowed Values UnaryTests](https://issues.omg.org/browse/DMN15-135)
Proposal: [DMN15-136 Align the usage of UnaryTests with decision table inputs](https://issues.omg.org/browse/DMN15-136)

Add typeConstraint to ItemDefinition

### CODE REVIEW

**Comment by falko** [comment]
@SimonRinguette can you give me push access to your branch? I've rebased it locally and would like to push to the PR.

**Comment by SimonRinguette** [comment]
@falko I think i gave you access properly, let me know if that worked

File: xmi/DMN15.xmi [hunk 1]
```diff
@@ -1,2565 +1,2666 @@
...
+                          name="DMNDI">
+            <ownedAttribute xmi:type="uml:Property" xmi:id="_17_0_5_1_a250249_1518651855436_638563_4054"
+                            visibility="private"
+                            type="_17_0_3_1_42401a5_1375643567471_779757_3017"
+                            association="_17_0_5_1_a250249_1518651855436_503672_4052"/>
+         </packagedElement>
+      </packagedElement>
+   </uml:Model>
+   <StandardProfile:Metaclass xmi:id="_17_0_2_3_ea50349_1446066806407_401082_5765"
+                              base_Class="_17_0_2_3_ea50349_1446066767165_84469_5739"/>
```
[the remark this answers is not in the document]
Comment (SimonRinguette) [reply xmi/DMN15.xmi:2665]: No... i can retry to generate it but i'm never certain of which options are used for the export

[the remark this answers is not in the document]
Comment (falko) [reply xmi/DMN15.xmi:2665]: I'll take it in for now and if needed we'll fix it later. The AB usually checks the XMI files.

**Pull request merged**